storage/src/adb/any/variable/mod.rs

```diff
@@ -96,35 +87,19 @@ pub struct Any<E: RStorage + Clock + Metrics, K: Array, V: Codec, H: CHasher, T:
     /// `log`.
     mmr: Mmr<E, H>,
 
-    /// A (pruned) log of all operations applied to the db in order of occurrence. The position of
-    /// each operation in the log is called its _location_, which is a stable identifier. Pruning is
-    /// indicated by a non-zero value for `pruned_loc`, which provides the location of the first
-    /// operation in the log.
+    /// A log of all operations applied to the db in order of occurrence. The _location_ of an
+    /// operation is its position in this log, and corresponds to its leaf number in the MMR.
```

Comment:
"leaf number" is an obsolete term, replaced by "location" of the leaf (see mmr/mod.rs). I think you can just drop the mention of MMR here, as this property is explained in the invariant below.

I think it's still useful here to note the log may be pruned, but I guess we can add that back in when we fix all the pruning boundary stuff later.

Reply:
Dropped reference to MMR; added "pruned" back.